front/app/components/UI/QuillEditor/index.tsx

```diff
@@ -26,10 +26,10 @@ export interface Props {
   value?: string;
   label?: string | JSX.Element | null;
   labelTooltipText?: string | JSX.Element | null;
-  noToolbar?: boolean;
```

Comment:
Not related to the links but I saw this is unused while I was making the types of the other noX props narrower.